src/resources/projects/controller.js

```diff
@@ -9,6 +9,33 @@ const getAllProjects = async (req, res) => {
     res.status(500).json({ error: error.message });
   }
 };
+
+const getProjectById = async (req, res) => {
+  /* 
+  Resources:
+  unary plus operator => https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Unary_plus 
+  */
+  const targetId = +req.params.id;
```

Comment:
Very nice! This is great when working with ints in a string but be careful about using it with floats.

`parseInt` is faster, so keep that in mind as well if you have any performance issues down the line as you start using this more :)

Reply:
We will, thank you! :)